### ISSUE TITLE
The Temporal Component of the Research

### ISSUE THREAD

**Issue by xiuyuanzhang** [uchicago-computation-workshop/luis_bettencourt#7]
The priority survey is a very powerful illustration of the inequality existing in countries and the urgent states of people's living condition. 

As it is already mentioned by Dr. Bettencourt and others in the Discussion and Outlook, the third concern of this set of data is, while the data itself has the characteristics of being consistent and complete, the time element raises some concerns. 

Perhaps with the scaling fit, one could take a look at Table.2 and see the difference in the heterogeneity index b from areas in Brazil and target area in South Africa. However, I am still wondering about the commensurability of these two numbers, more specifically, how applicable is a comparison like this to a potential implementation of social welfare improvement projects?

Might it be more informative to the mappings of sustainability developments if it is the comparison between data from different years but with the same regions in one country? From the comparison between South Africa and Brazil, the difference between their status of development is apparent. Nevertheless, it does not seem to be a comparison that directly suggests replicability. Each country has unique circumstances both in geography and the needs (priority) of their populations. Admittedly, collecting data this large is extremely difficult, but ideally it would seem like two sets of data for the same country, a certain time interval apart -with a survey that is conducted twice, respectively with those collections of data- would give more information and attract more people to invest in these kind of sustainability projects.

Reference: Heterogeneity and scale of sustainable development in cities.

**Comment by lmbett** [comment]
Yes, the comparison over time needs to be done. In some cases we have data, in some other cases it is still coming... but it will be there soon.
